doc/api.rst

```diff
@@ -113,6 +113,22 @@ Indexing
    Dataset.reset_index
    Dataset.reorder_levels
 
+Missing value handling
+----------------------
+
+.. autosummary::
+   :toctree: generated/
+
+   Dataset.isnull
+   Dataset.notnull
+   Dataset.count
+   Dataset.dropna
+   Dataset.fillna
+   Dataset.ffill
+   Dataset.bfill
+   Dataset.interpolate_na
+   Dataset.where
+
```

Comment:
I think it would be nice if we could see docstrings for these methods in rtd.

Reply:
Works for me, but these should be removed from `api-hidden.rst` now.

Reply:
Done.